.github/workflows/event-merge-to-queue.yml

```diff
@@ -36,10 +35,10 @@ jobs:
     uses: ./.github/workflows/flow-test.yml
     secrets: inherit
     with:
-      platform: all
+      platform: 'ubuntu:default,macos,alpine:3'  # on feature branches this should be 'all'
```

Comment:
Thinking about it, we should also drop `macos-15-intel` from the matrix. It's the least relevant OS in the matrix (as Apple doesn't have modern x86 machines, and it's going to be the last runner GH will supply). It's also only of the longer runs we have